src/xcode/ENA/ENA/Source/Extensions/String+Today.swift

```diff
@@ -17,12 +17,22 @@
 
 import Foundation
 
+private var todayFormatter = DateFormatter.packagesDateFormatter
```

Comment:
One minor brain fart regarding 6c87f8ea9dac79fbf337d1f8ffc83560f3ec8a57:

```swift
extension DateFormatter {
    static var packagesDateFormatter: DateFormatter = {
        print("Init formatter")
        let formatter = DateFormatter()
        formatter.dateFormat = "yyyy-MM-dd"
        formatter.timeZone = TimeZone(abbreviation: "UTC")
        formatter.locale = Locale(identifier: "en_US_POSIX")
        formatter.calendar = Calendar(identifier: .gregorian)

        return formatter
    }()
}
```

This also makes sure the `packagesDateFormatter` will only be instantiated once and you don't need to have it stored in this `private var`.